### PR TITLE
Add example tests

### PR DESCRIPTION
**Pull request by Awarua-**
- Add example tests
- Include translation example
- Add gitignore

### CODE REVIEW

File: test/acceptance/hello.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+var assert = require('chai').assert,
```
Comment (drkno) [inline test/acceptance/hello.js]: ES6 for consistency with the rest of the Concierge codebase (you'll probably need 'use strict' at the top due to grunt + node4)

var -> const/let
function -> arrow

**Review by drkno — CHANGES_REQUESTED** [review]

File: test/acceptance/hello.js (hunk 1 above)
Comment (Awarua-) [reply test/acceptance/hello.js]: done

Comment (drkno) [reply test/acceptance/hello.js]: ... but we still have `function`s ...

File: test/unit/test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+var assert = require('chai').assert;
```
Comment (drkno) [inline test/unit/test.js]: and vars

**Review by drkno — APPROVED** [review]

**Pull request merged**